### ISSUE TITLE
Server display IP issue

### ISSUE THREAD

**Issue by Saqib117** [billycao/Wi-Mote#7]
Currently showing wrong IP address in server,

I already mentioned to James but just wanted to put it on the issues... i thought it was here but i guess not.

**Comment by billycao** [comment]
Just a note, it actually shows the right IP but just picks from the default network adapter. I believe fixing this problem would involve letting the user pick the network adapter or have a smarter way of picking one.

**Comment by Saqib117** [comment]
yeah, but from the user point of view, they shouldn't have to deal with that :-/ it shows my hamachi IP, maybe an option to cycle through IPs? and then for that particular computer it just remember what IP in the sequence to show next time.

(from users point of view it would say something along the lines of "if this IP doesn't work cycle to next and try again")

**Comment by billycao** [comment]
I don't like the idea of cluttering the UI with text. I rather like the UX principle of simplicity. I don't know how the server currently decides which network adapter to use, but I think, if possible, the optimal would be picking the adapter in a smarter way (I'm sure James got this), and then allow the user to change the adapter in settings.

In the end I trust James' decision on this.
